calico_cni/policy_drivers.py

```diff
@@ -131,6 +131,14 @@ def generate_tags(self):
         """
         return []
 
+class MesosPolicyDriver(DefaultPolicyDriver):
+    def generate_rules(self):
+        allow = Rule(action="allow")
+        rules = Rules(id=self.profile_name,
+                      inbound_rules=[allow],
```

Comment:
There is no policy-on / policy-off switch for Mesos. I went with allow-all for a better demo story. 

I think the user would prefer standard mesos networking rules (everything talks to everything) and then lock down their app from there.